Resources/ProtobufDumper/ProtobufDumper/ImageFile.cs

```diff
@@ -668,7 +680,25 @@ private string GetValueForProp(dynamic propInfo, object options, bool field, out
                         string value = GetValueForProp(fieldInfo, deserialized, true, out name);
 
                         if (value != null)
-                            options_kv.Add(name, value);
+                        {
+                            if (options_kv.ContainsKey(name))
+                            {
+                                if (options_kv[name] == value)
+                                {
+                                    Console.WriteLine("Found a duplicate option '{0}' with same value, ignoring...", name);

```

Comment:
I'm curious if the serialized proto descriptors actually include duplicate options in their byte steams, or if this is a bug elsewhere in protobuf dumper.